--- origsrc/gtkpod-0.99.12/configure.in	2007-12-17 22:37:46.000000000 -0600
+++ src/gtkpod-0.99.12/configure.in	2007-12-22 19:36:39.921875000 -0600
@@ -190,8 +190,8 @@
 dnl Check for libvorbisfile
 AC_ARG_WITH(ogg, AC_HELP_STRING([--without-ogg], [Disable Ogg/Vorbis support]))
 if test "x$with_ogg" != "xno"; then
-    AC_SEARCH_LIBS(ov_open, ["vorbisfile"],
-        [AC_CHECK_HEADER(vorbis/vorbisfile.h, have_ogg=yes, have_ogg_header=no)], have_ogg=no)
+    AC_CHECK_LIB(vorbisfile, ov_open,
+        [AC_CHECK_HEADER(vorbis/vorbisfile.h, LIBS="-lvorbisfile -lvorbis $LIBS" have_ogg=yes, have_ogg_header=no)], have_ogg=no)
     if test "x$have_ogg_header" = "xno"; then
         have_ogg="no"
         AC_MSG_WARN([vorbis/vorbisfile.h cannot be found or is unusable. Check your ogg/vorbis installation.])
